### DISCUSSION TITLE
Is there a font that supports the C++ prompt?

### DISCUSSION THREAD

**Discussion by ryanpcmcquen** [starship/starship#1786] [Q&A]
I've tried quite a few of the available nerd fonts, including Fira Code, but none of them seem to have the glyph for C++ projects:
<img width="650" alt="Screen Shot 2020-10-13 at 10 05 06 AM" src="https://user-images.githubusercontent.com/772937/95892580-980aac80-0d3b-11eb-908d-de20c3bad5af.png">

Does anyone know of one?

**Comment by ryanpcmcquen** [comment]
Actually, I think this is CMake, and DaddyTimeMono supports it:
<img width="723" alt="Screen Shot 2020-10-13 at 10 26 24 AM" src="https://user-images.githubusercontent.com/772937/95895420-7c090a00-0d3f-11eb-88f6-c9e023d37dce.png">

**Comment by matchai** [comment]
It looks like the symbol used has little font support and will need to be changed.
It is supposed to look like this:
![image](https://user-images.githubusercontent.com/4658208/95905359-c18cfd80-0d66-11eb-81cf-30577175a93c.png)

**Reply by ryanpcmcquen** [reply]
@matchai, what font is that??

**Reply by matchai** [reply]
I stole the screenshot from the PR where the feature was introduced 😅 
@ATiltedTree should be able to answer.

In the meantime, the following PR should address the issue across fonts: https://github.com/starship/starship/pull/1787
